src/mpid/ch4/netmod/ofi/ofi_pre.h

```diff
@@ -171,6 +172,7 @@ typedef struct {
     uint64_t win_id;
     struct MPIDI_OFI_win_request *syncQ;
     MPIDI_OFI_win_targetinfo_t *winfo;
+    struct MPIDI_OFI_win_acc_hint *acc_hint;
```

Comment:
Why is this dynamically allocated?

Reply:
Seems unnecessary..Will fix

Reply:
Hmm, okay, then sounds like this is what we have to do anyway. Let's document on why this is a pointer as a comment here.

Reply:
Done